tests/foreman/cli/test_remoteexecution.py

```diff
@@ -138,16 +137,16 @@ def test_positive_run_default_job_template_by_ip(self, fixture_vmsetup):
             )
             raise AssertionError(result)
 
-        task = Task.list_tasks({"search": command})[0]
-        search = Task.list_tasks({"search": 'id={}'.format(task["id"])})
-        assert search[0]["action"] == task["action"]
+        task = Task.list_tasks({'search': command})[0]
+        search = Task.list_tasks({'search': f'id={task["id"]}'})
+        assert search[0]['action'] == task['action']
 
     @pytest.mark.skip_if_open('BZ:1804685')
     @pytest.mark.tier3
     @pytest.mark.pit_client
     @pytest.mark.pit_server
-    @pytest.mark.parametrize('fixture_vmsetup', [{'nick': 'rhel7'}], ids=['rhel7'], indirect=True)
-    def test_positive_run_job_effective_user_by_ip(self, fixture_vmsetup):
+    @pytest.mark.rhel_ver_list([7])
```

Comment:
Is there any reason for not including other versions here? I'd suggest at least `[7,8,9]`